var/spack/repos/builtin/packages/py-celery/package.py

```diff
@@ -46,19 +47,26 @@ class PyCelery(PythonPackage):
     # 'cosmosdbsql',
     # 's3',
 
+    depends_on('python@3.7:', type=('build', 'run'), when="@5.2.3:")
     depends_on('python@2.7:2.8,3.4:', type=('build', 'run'))
 
     depends_on('py-setuptools', type=('build', 'run'))
 
     depends_on('py-redis@3.2.0:', when='+redis', type=('build', 'run'))
     depends_on('py-sqlalchemy', when='+sqlalchemy', type=('build', 'run'))
 
-    depends_on('py-click@7.0:7', when="@5.0.0:5", type=('build', 'run'))
+    depends_on('py-click@7.0:7', when="@5.0.0:5.0", type=('build', 'run'))
+    depends_on('py-click@8.0.3:8', when="@5.2.0:5.2", type=('build', 'run'))
     depends_on('py-click-didyoumean@:0.0.3', when="@5.0.0:5", type=('build', 'run'))
```

Comment:
At least in this latest version, this should be `@0.0.3:`

Reply:
I'll have an update posted shortly.